yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -649,3 +649,42 @@ func TestRegexFunctionsIssue(t *testing.T) {
 	}
 
 }
+
+func TestCopyUnsupportedConstructIssuesDetected(t *testing.T) {
```

Comment:
see if the tests for this can be at one place only either in detectors.go or parse_issue_detector_test.go

Currently we have the tests distributed among these two files, but since they are essentially testing the same thing, we can just have it one place.

Reply:
Yeah for now I have added the tests but commented out the sql of CREATE ... ON_ERROR since it is supported only on PG17. Have added a ticket to do the same:
[https://yugabyte.atlassian.net/browse/DB-14671?atlOrigin=eyJpIjoiMTg4ZDE3NDI2MWM4NGI1MmExYzE4YzgzNjk3YjczMmMiLCJwIjoiaiJ9](url)

Reply:
Removed the tests from detectors_test.go. The tests are same anyways,